### ISSUE TITLE
API Request: Static CSS Selector for style overrides

### ISSUE THREAD

**Issue by peterjcaulfield** [pmndrs/leva#111]
It would be good to assign a static css selector so that consumers can apply css overrides to the leva pane.

Right now the only css classes are generated classes which makes it impossible to override certain styles.

My use case is to adjust to initial position of a global pane.

**Comment by dbismut** [comment]
Leva uses the styling library stitches. You can easily adjust the theme variables through the `theme` prop. Have a look at this sandbox:

https://codesandbox.io/s/leva-theme-9v0ko

Regarding the initial position, it's not customizable yet but I'll work on something.

**Comment by peterjcaulfield** [comment]
Yep I noticed that but in some cases every style declaration wont be part of the theme (or you may want a localised override) so in these cases an "escape hatch" is needed.

I think the general approach is good though 👌🏼
